recipes/jupyterlab-day/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "jupyterlab-day" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f9e2e9ba825081905c422582d8d9bb945c5045fa08d3f4a1513d72614f1f9ac7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python  >=3.6
+    - pip
+    - jupyterlab
+    - yarn
+    - jupyter-packaging
+  run:
+    - python  >=3.6
```

Comment:
Probably should have `jupyterlab` for run, no?

Also, FWIW upstream lists `python >=3.7`.

Reply:
Thanks for the review!